packages/amplication-client/src/Settings/ApiTokenList.tsx

```diff
@@ -78,11 +78,9 @@ export const ApiTokenList = React.memo(() => {
         </Button>
       </div>
       <div className={`${CLASS_NAME}__message`}>
-        API tokens are used to authenticate requests to Amplication API,
-        specifically it is required to use Amplication CLI.
+        API tokens are used to authenticate requests to the Amplication API. They are specifically required for Amplication CLI.
```

Comment:
```suggestion
        API tokens are used by the Amplication CLI to make requests to the API.
```
I think in this case, less is more.